config/manager/enoDeployment.yaml

```diff
@@ -27,13 +27,21 @@ spec:
         - /manager
```

Comment:
The docker file already has an entry point running the binary "manager", is this required?

Reply:
As I said in the meeting. There is no harm to specify the command also in the deployment. At least this is the way that operator-sdk generated the code so I have decided to not change that part and keep it simple. The command field in pod will override the Entrypoint in the container. Below you can find the info from K8S docs:

"If you supply a command and args, the default Entrypoint and the default Cmd defined in the Docker image are ignored. Your command is run with your args."

https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#notes